tests/dummy/app/pods/checkbox/template.hbs

```diff
@@ -1,172 +1,154 @@
-<div class="section">
-  <div class="title">frost-checkbox</div>
-  <div class="example">
-    <div class="title">Basic</div>
-    <div class="snippet">
-{{format-markdown "```handlebars
-{{frost-checkbox id='basic'}}
-```"
-}}
+<div class="dummy-notification-container">
+{{#each notifications as |notification|}}
+  {{notification-message notification=notification}}
+{{/each}}
+</div>
+
+<div class='dummy-body text-input field'>
```

Comment:
@vesper2000 the t'ext-input field' class defines a width on a per component basis to help set a max width for all the snippet blocks. So you can add one for checkbox based on what a good width would be for them